sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache_impl.hpp

```diff
@@ -71,7 +62,7 @@ double joint_matmul(TOperand *A, TOperand *B, TResult *C, queue &q, int i) {
   assert(colsA == rowsB);
   assert(rowsA % tM == 0);
   assert(colsA % tK == 0);
-  assert(colsB % tN == 0);
+  assert(colsB % TN == 0);
```

Comment:
why upper case here?

Reply:
Thought to unify with other tests, where I'm making change (since most of other tests are using TN), but if you believe it is better to keep it tN, I can change that back.

Reply:
Yes, I think it should match the other tM and tK. Unless if you also change to TM and TK, in this case, we can have TN

Reply:
done